config.json

```diff
@@ -1441,6 +1441,18 @@
       "difficulty": 0,
       "topics": null,
       "deprecated": true
+    },
+    {
+      "slug": "leap",
+      "uuid": "7c8294ee-5924-4bf8-a72f-31d0e2d7d9a0",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
```

Comment:
According to policies, exercises with difficulty level of 1 also need to provide a hint file referencing the hello world tutorial. Please add this file.

Reply:
What should be the contents of hint file? I cannot think of a good hint for this problem.

Reply:
It is described [here](https://github.com/exercism/java/blob/master/POLICIES.md#reference-tutorial-in-the-first-exercises).

Reply:
Updated hints.md file